### ISSUE TITLE
V5 Beta Option   arrowKeys: false,  has no effect

### ISSUE THREAD

**Issue by acwolff** [dimsemenov/PhotoSwipe#1794]
I used this code:

    	import PhotoSwipeLightbox from './res/photoswipe-lightbox.esm.js'; 
    	import PhotoSwipeDynamicCaption from './res/photoswipe-dynamic-caption-plugin.esm.js';
    	const options = {
    	  arrowKeys: false,
    	  gallerySelector: '#gallery',

I did expect that the arrow buttons would disappear, but they are still visible.

Do I something wrongly or is this a bug?

**Comment by BilouMaster** [comment]
`arrowKeys` is an option to enable or disable keyboard navigation.

To remove the arrow buttons, try:
```
arrowPrev: false,
arrowNext: false
```

If it doesn't work, try:
```
arrowPrevSVG: '',
arrowNextSVG: ''
```

**Comment by acwolff** [comment]
> To remove the arrow buttons, try:
> 
> ```
> arrowPrev: false,
> arrowNext: false
> ```
Yes, that works, but in that case I can't use next code to solve the [right arrow problem](https://github.com/dimsemenov/PhotoSwipe/issues/1791):

    lightbox.on('change', () => {
        document.querySelector('.pswp__button--arrow--next').style.zIndex = "9999";
    });

 So I will use the method  I did give above.

**Comment by BilouMaster** [comment]
> > To remove the arrow buttons, try:
> > 
> > ```
> > arrowPrev: false,
> > arrowNext: false
> > ```
> Yes, that works too, but in that case I can't use next code to solve the [right arrow problem](https://github.com/dimsemenov/PhotoSwipe/issues/1791)

If you don't want to show the arrows... There is no more issue to solve with superposition and you don't need that code anymore, right? ... 🤔

**Comment by acwolff** [comment]
Correct, but some of my users like to hide these buttons on mobile devices, but not on PC's, so in that case I have still to solve the issue.

**Comment by BilouMaster** [comment]
Yup, but we still can hide the ui elements with a single tap on screen. :) I like that feature

**Comment by acwolff** [comment]
Aha, I did not know that.

Thanks for this tip!
